README.md

```
@@ -83,7 +84,11 @@ You can find our docs [online][clp-docs] or view the source in `docs/src`.
 
 You can use GitHub issues to [report a bug][bug-report] or [request a feature][feature-req].
 
-Join us on [Zulip][zulip] to chat with developers and other community members.
+# Community
+Need help? Join us on one of our community servers:
+
+* [![Discord][discord-badge]][yscope-community-discord]
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: dash; Actual: asterisk

(MD004, ul-style)